### ISSUE TITLE
function optionally shown at call site

### ISSUE THREAD

**Issue by psifertex** [Vector35/binaryninja-api#1621]
It would be nice if function tags were shown (or optionally shown behind a setting or tag property, preference toward the former) at the call site in addition to the function graph view basic block.

Related so I'm not going to file another ticket: Linear view doesn't show function tags in the function prototype.

**Comment by galenbwill** [comment]
> Related so I'm not going to file another ticket: Linear view doesn't show function tags in the function prototype.

Is that still true?

**Comment by psifertex** [comment]
Nope, linear view prototype does show it now. Just the original issue left. Though the function header at the top of graph view doesn't (though it's arguable whether it should)

**Comment by galenbwill** [comment]
@psifertex ftfy ;-)
